### PR TITLE
feat: add tslib import transform

### PR DESCRIPTION
**Pull request by filipesilva**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "loader-utils": "^1.1.0",
     "magic-string": "^0.19.1",
     "source-map": "^0.5.6",
-    "typescript": "^2.3.2"
+    "typescript": "^2.4.0-dev.20170608"
```
Comment (IgorMinar) [inline package.json:22]: why do we need this version? rolling this out while it depends on a nightly build of typescript would be very problematic.

File: src/import-tslib.ts [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+function createTslibImport(node: ts.Node): ts.Node {
+  const name = getVariableStatementName(node);
+
+  if (!name) {
+    return node;
+  }
+
+  const namedImports = ts.createNamedImports([ts.createImportSpecifier(undefined, ts.createIdentifier(name))]);
+  // typescript@next is needed for a fix to the function parameter types of ts.createImportClause.
+  // https://github.com/Microsoft/TypeScript/pull/15999
```
Comment (IgorMinar) [inline src/import-tslib.ts:35]: aha!

File: package.json (hunk 1 above)
Comment (IgorMinar) [reply package.json:22]: nvm, I see the comment below.

**Review by IgorMinar — APPROVED** [review]
lgtm, except that you should also update the version in package.json otherwise you will confuse yarn/npm by changing the content but not version number.

**Comment by filipesilva** [comment]
Bumped version number and merged.

**Pull request merged**